function/src/main/proto/cloudevents.proto

```diff
@@ -0,0 +1,71 @@
+/**
+ * CloudEvent Protobuf Format
+ *
+ * - Required context attributes are explicity represented.
+ * - Optional and Extension context attributes are carried in a map structure.
+ * - Data may be represented as binary, text, or protobuf messages.
+ */
+
+syntax = "proto3";
+
+package io.cloudevents.v1;
+
+//import "google/protobuf/any.proto";
```

Comment:
The `any.proto` and `timestamp.proto` were taken directly from `google/protobuf` and copied in here. I am not sure if there is a better recommended way to do this, but this is what I went w/ for now. Feels like there is probably a better way.

Reply:
I think the recommended approach is to include `google/protobuf/any.proto` etc. on the `protoc` command line. I think we could do that with the Maven plugin as well with some configuration.

Reply:
Cool. I will try to do this next .